src/main/java/seedu/task/model/task/Task.java

```diff
@@ -85,6 +87,9 @@ public UniqueTagList getTags() {
      * Replaces this tasks's tags with the tags in the argument tag list.
      */
     public void setTags(UniqueTagList replacement) {
+        //@@evanyeung A0163744B
+        assert replacement != null;
```

Comment:
It means you can't use a null replacement. The `UniqueTagList` can still have no tags in it